tutorials/plot_stats_cluster_erp.py

```diff
@@ -113,18 +113,7 @@
                   topomap_args=time_unit)  # show difference wave
 
 # Create ROIs by checking channel labels
-pos = find_layout(epochs.info).pos
-rois = dict()
-for pick, channel in enumerate(epochs.ch_names):
-    last_char = channel[-1]  # for 10/20, last letter codes the hemisphere
-    roi = ("Midline" if last_char in "z12" else
-           ("Left" if int(last_char) % 2 else "Right"))
-    rois[roi] = rois.get(roi, list()) + [pick]
-
-# sort channels from front to center
-# (y-coordinate of the position info in the layout)
-rois = {roi: np.array(picks)[pos[picks, 1].argsort()]
-        for roi, picks in rois.items()}
+rois = create_1020_montage_mapping(evoked.info, midline="12z")
```

Comment:
fractions of channels are called "selection" in MNE jargon. To be consistent here we have a

make_channel_selections(info, ....)

see that there something like this with the group_by and order params in raw.plot

Reply:
`make_1020_channel_ROI_selection`?